### ISSUE TITLE
Connect: Talk to HSA without waiting on hold

### ISSUE THREAD

**Issue by alanjosephwilliams** [codeforamerica/project-ideas#38]
**BLUF**: SNAP recipients, applicants and assisters (like food bank staff) frequently need to call HSA directly to learn the status of an individual's benefits. There is a high volume of calls relative to staff available to take them, so hold times can be quite long—on the order of a half hour to an hour oftentimes.

In 2014, Twilio robots _can wait on hold for you_. Let's give recipients, applicants and assisters the ability to initiate a phone call to the Human Services Agency via SMS, and hold their place in the queue. When staff are available and take the call of hold, we can play a short message explaining that someone is waiting, and connect the two parties by calling the end-users back.

The interaction flow would be something like:
- User initiates a call to HSA via text message. They send an SMS reading "connect" to a dedicated number.
- Twilio initiates a call to HSA, and enters any information required by a phone tree by simulating the appropriate keypress.
- When an Eligibility Worker accepts the inbound call from Twilio, we play a brief recorded message stating that a caller is waiting, and that they will be connected momentarily. 
- Twilio then initiates a call to the phone number from which it received the text message, connecting the Eligibility Worker to the user.

Violá! We will have reduced the time the user or assister spends trying to connect to staff, eliminated the tension and uncertainty that comes with waits of indeterminate length, and hopefully given the end user the ability to do something else productive with their most precious resource.

**Comment by lippytak** [comment]
I LOVE LOVE LOVE this idea. Here are some notes from our first two (potential users) at the Food Bank:
![liliana](https://cloud.githubusercontent.com/assets/2533112/4100505/83a107ca-309d-11e4-9cb8-76845b08d3d1.png)
![cindy](https://cloud.githubusercontent.com/assets/2533112/4100506/b8e00f3a-309d-11e4-9932-7f8761953540.png)

**Comment by lippytak** [comment]
And here's the more generalized **BLUF** to stoke @daguar's enthusiasm: Poor people have to wait on hold a lot. Rich people don't.

As always, back to Ezra Klein:

> one privilege the insured and well-off have is to excuse the terrible quality of services the government routinely delivers to the poor. Too often, the press ignores -- or simply never knows -- the pain and trouble of interfacing with government bureaucracies that the poor struggle with daily. That can allow the problems in those bureaucracies to fester - [wapo](http://www.washingtonpost.com/blogs/wonkblog/wp/2013/10/25/obamacares-problems-go-much-deeper-than-the-web-site/)

Hopefully we can track down avg hold times and abandoned call rates for various gov services.

**Comment by daguar** [comment]
Reminds me at some point to bug Ezra K about our work...

**Comment by RebeccaCoelius** [comment]
Do you know Ezra? I have a bunch of connections if would be helpful.

Id like to +1000 to this one...

On Sunday, August 31, 2014, Dave Guarino notifications@github.com wrote:

> Reminds me at some point to bug Ezra K about our work...
> 
> > On Aug 30, 2014, at 4:40 PM, Jake Solomon <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > 
> > And here's the more generalized BLUF to stoke @daguar's enthusiasm: Poor
> > people have to wait on hold a lot. Rich people don't.
> > 
> > As always, back to Ezra Klein:
> > 
> > one privilege the insured and well-off have is to excuse the terrible
> > quality of services the government routinely delivers to the poor. Too
> > often, the press ignores -- or simply never knows -- the pain and trouble
> > of interfacing with government bureaucracies that the poor struggle with
> > daily. That can allow the problems in those bureaucracies to fester - wapo
> > 
> > Hopefully we can track down avg hold times and abandoned call rates for
> > various gov services.
> > 
> > —
> > Reply to this email directly or view it on GitHub.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/codeforamerica/health-project-ideas/issues/38#issuecomment-53976903
> .

## 

Rebecca Coelius, MD
Code for America
415 298 2872
LinkedIn http://www.linkedin.com/in/rebeccacoelius
Twitter http://twitter.com/#!/RebeccaCoelius

**Comment by fureigh** [comment]
So so so good!

Has a staff member already been secured for testing their side of the experience?

**Comment by lippytak** [comment]
It's aliiiiiiive: https://github.com/codeforamerica/connect/issues/9

**Comment by daguar** [comment]
Belated note, but this is now in a rebuilt alpha form here: https://github.com/codeforamerica/connect

We're actively testing with a small set of early adopter users to determine the connection success rate.
